### DISCUSSION TITLE
TgDownloader Winx64 usage

### DISCUSSION THREAD

**Discussion by wtfismyname69420** [DamianMorozov/OpenTgResearcher#40] [Q&A] [answered]
Hello! This is a great software you've written and I commend you! 
But there is no proper usage documentation for the GUI version and it seems I am unable to download files!

**Comment by selimsum** [comment]
Indeed, seems like it's impossible to connect to TG server when starting from scratch.

**Comment by DamianMorozov** [comment, marked as answer]
Try to use this [Guide to setting up the desktop version](https://github.com/DamianMorozov/TgDownloader/blob/main/GUIDE-SET-DESKTOP.md#guide-to-setting-up-the-desktop-version).

**Reply by selimsum** [reply]
Unfortunately, it doesn't help. I tried clicking connect after filling in API hash, API ID  and phone number, but nothing happens. No error message is displayed. Maybe it's a bug?

What worked for me was to create the session file in the console version and copying it over.

**Reply by DamianMorozov** [reply]
Check new update [v0.2.550](https://github.com/DamianMorozov/TgDownloader/releases/tag/v0.2.550).
That should solve your problem.
